src/users/dto/userGetDto.ts

```diff
@@ -1,9 +1,16 @@
-import { User } from "../entity/user";
+import { UserPivot } from "../entity/user";
+import { pivotGetDto } from "../../events/dto/pivotGetDto";
 
-export const UserGetDto = (user: User): User => ({
-  id: user.id,
-  firstName: user.firstName,
-  lastName: user.lastName,
-  email: user.email,
-  events: user.events,
-});
+export const userGetDto = (user: UserPivot): UserPivot => {
+  return {
+    id: user.id,
+    firstName: user.firstName,
+    lastName: user.lastName,
+    email: user.email,
+    rating: user.rating,
+    performerType: user.performerType,
+    pivot: user?.pivot?.length
+      ? user.pivot.map((pivot) => pivotGetDto(pivot))
+      : user.pivot,
```

Comment:
What if the pivot will be undefined/null? We should send it to FE? I think empty Array approach would be cleaner